PowerEditor/installer/nativeLang/estonian.xml

```diff
@@ -1,903 +1,1008 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!--
-*Estonian language file for Notepad++*
-
-	** Created By:- Yusuf Mullaji *
-		Email Id-> yusufmullaji@rocketmail.com
-
+Estonian translation by Andres Traks
+https://github.com/AndresTraks/
```

Comment:
"Contributor" means someone who has contribute somethings.
Automatic translation brings bad user experience, so I would say that not only you have done nothing for the contribution but also did somethings bad to the project. 
I don't see in any case why you deserve to be named in Notepad++ project.